vcf-dumper/vcf-dumper-lib/src/main/java/uk/ac/ebi/eva/vcfdump/VariantExporterController.java

```diff
@@ -319,24 +244,17 @@ private VariantContextWriter buildVcfFileWriter() {
         outputFilePath = Paths.get(outputDir).resolve(fileName);
 
         VariantContextWriterBuilder builder = new VariantContextWriterBuilder();
-        VariantContextWriter writer = builder.setOutputFile(outputFilePath.toFile())
-                                             .unsetOption(Options.INDEX_ON_THE_FLY)
-                                             .build();
-        return writer;
+        return builder.setOutputFile(outputFilePath.toFile()).unsetOption(Options.INDEX_ON_THE_FLY).build();
     }
 
     private VariantContextWriter buildVcfOutputStreamWriter() {
         VariantContextWriterBuilder builder = new VariantContextWriterBuilder();
-        VariantContextWriter writer = builder.setOutputVCFStream(outputStream)
-                                             .unsetOption(Options.INDEX_ON_THE_FLY)
-                                             .build();
-        return writer;
+        return builder.setOutputVCFStream(outputStream).unsetOption(Options.INDEX_ON_THE_FLY).build();
     }
 
-    private Set<String> getChromosomes(QueryOptions options) {
+    private Set<String> getChromosomes(List<String> regions) {
         Set<String> chromosomes;
 
-        List<String> regions = options.getAsStringList(VariantDBAdaptor.REGION);
         if (regions.size() > 0) {
             chromosomes = getChromosomesFromRegionFilter(regions);
         } else {
```

Comment:
We should consider adding the distinct query to VariantsWithAnnotationsService so we can remove the Cellbase dependency also. We are importing lot of new species from dbSNP now and probably some of them are not in Cellbase

Reply:
We are not querying cellbase, but EVA web services. We should rename 'cellbaseClient' to 'evaWsClient' then.